MediaBrowser.Api/PluginService.cs

```diff
@@ -243,9 +245,7 @@ public async Task Post(UpdatePluginConfiguration request)
             // https://code.google.com/p/servicestack/source/browse/trunk/Common/ServiceStack.Text/ServiceStack.Text/Controller/PathInfo.cs
             var id = Guid.Parse(GetPathValue(1));
 
-            var plugin = _appHost.Plugins.First(p => p.Id == id) as IHasPluginConfiguration;
-
-            if (plugin == null)
+            if (!(_appHost.Plugins.FirstOrDefault(p => p.Id == id) is IHasPluginConfiguration plugin))
```

Comment:
Assign the value returned by `FirstOrDefault` first to a variable, this is just a tad to long to have inline

Reply:
Fixed